schemas/context/namespace.description.md

```diff
@@ -0,0 +1,15 @@
+The namespace ensures first, second, and third party user and device identities can be used in customer engagement platforms.
+
+Namespaces are used to attain global uniqueness with user and device identities that originate in distributed, uncoordinated identity environments. 
```

Comment:
The claim that they create global uniqueness is not backed up by the description. We do have a mechanism for ensuring global uniqueness (URIs) that for some reason that is not made clear is not being used here.

I can assume that the reason is that URIs might be a bit too long as a wire format, but this is only an assumption (and my assumptions have been false in the past)